requirements.txt

```diff
@@ -118,4 +119,8 @@ wcwidth==0.2.8
 webcolors==1.13
 webencodings==0.5.1
 websocket-client==1.6.4
-widgetsnbextension==4.0.9
\ No newline at end of file
+widgetsnbextension==4.0.9
+pip~=22.3.1
```

Comment:
pip dependencies should not be included in requirements.txt; this also applies to wheel and setuptools

Reply:
Removed